x-pack/plugins/osquery/public/components/accordion.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import styled from 'styled-components';
+import { EuiAccordion } from '@elastic/eui';
+
+export const StyledEuiAccordion = styled(EuiAccordion)`
```

Comment:
Look like this `StyledEuiAccordion` was extracted out but wasn't used for replacing any of its current uses?

Reply:
Good catch, there was quite a big refactor in the meanwhile, I'll remove this file 👍